gpameter.user.js

```diff
@@ -27,9 +27,15 @@ var grade_values = {
 };
 console.log(studentId);
 add_checkboxes = function(){
+	var mapElems = new Map();
```

Comment:
How about naming this something like `courses_checked` instead of `mapElems` which doesn't give any information about what data this contains?

Also, please use snake_case instead of camelCase for consistency.